src/PLCTag.jl

```diff
@@ -72,26 +78,93 @@ module PLCTag
 		:name => tagName,
 	)), '&')
 	
-	# TODO: create Bool versions as well
+	
+	const PLCPrimitives = Union{
+		Bool,
+		Int8, UInt8, Int16, UInt16,
+		Int32, UInt32, Int64, UInt64,
+		Float32, Float64
+	}
+	
 	for T in (
+		:Bool,
 		:Int8, :UInt8, :Int16, :UInt16,
 		:Int32, :UInt32, :Int64, :UInt64,
 		:Float32, :Float64,
 	)
-		@eval plcget(::Type{$(T)}, args...; kwargs...) = C.$(Symbol("plc_tag_get_", lowercase(String(T))))(args...; kwargs...)
-		@eval plcset(::Type{$(T)}, args...; kwargs...) = C.$(Symbol("plc_tag_set_", lowercase(String(T))))(args...; kwargs...)
+		if T === Bool
+			# TODO: determine what "bool" size is on a PLC
```

Comment:
The size of a bool on the PLC varies for the following cases:
- independent tag
- part of an array
- part of a UDT (user defined type)

https://github.com/kyle-github/libplctag/issues/82

For this reason, libplctag does not have support for bools. Our solution is to ban the usage of bools on the PLC, and only use SINT (1 byte).